Eplant/UI/Sidebar.tsx

```diff
@@ -1,48 +1,70 @@
+import { useState } from 'react'
+import { useContext } from 'react'
+import { Sidebar } from 'react-pro-sidebar'
+import { Menu } from 'react-pro-sidebar'
```

Comment:
Where is this Menu being used? Does `react-pro-sidebar` do something we can't do with MUI?

Reply:
I removed the package now (this was just a package I was playing around with)